### ISSUE TITLE
CAPI Lagging at Florianz Deposits in HIP 41526

### ISSUE THREAD

**Issue by outsidefactor** [EDCD/EDMarketConnector#1198]
- Version: [5.1.1]
 - Game Version: ['Odyssey']
 - OS: [Linux Manjaro]
 - OS Locale: [English]
 - If applicable: Browser [chrome]
 - Please attach **BOTH** log fil
[EDMarketConnector-debug.log](https://github.com/EDCD/EDMarketConnector/files/6815603/EDMarketConnector-debug.log)
es, by dragging and dropping them into this input:
     1. `%TEMP%\EDMarketConnector.log` from *immediately* after the bug occurs (re-running the application overwrites this file).
     1. `%TEMP%\EDMarketConnector\EDMarketConnector-debug.log`.  See [Debug Log File](https://github.com/EDCD/EDMarketConnector/wiki/Troubleshooting#debug-log-files).  NB: If you don't have this log file then you're not running the latest version of the application and should update first to see if we already fixed the bug you're reporting.

**Describe the bug**
When docking at a particular station EDMC always reports "Error: Frontier server is lagging". Restarting the client and EDMC does not resolve. Happens predictably.

**To Reproduce**
1. Travel to HIP 41526
2. Dock at Florianz Deposits
3. See error

**Expected behavior**
Not to see an error

**Screenshots**

**Additional context**
[Journal.210714172115.01.log](https://github.com/EDCD/EDMarketConnector/files/6815627/Journal.210714172115.01.log)
[HIP 41526.Florianz Deposits ++.2021-07-14T21.10.zip](https://github.com/EDCD/EDMarketConnector/files/6815650/HIP.41526.Florianz.Deposits.%2B%2B.2021-07-14T21.10.zip)
[HIP 41526.Florianz Deposits ++.2021-07-14T21.13.zip](https://github.com/EDCD/EDMarketConnector/files/6815652/HIP.41526.Florianz.Deposits.%2B%2B.2021-07-14T21.13.zip)

**Comment by Athanasius** [comment]
This is because the CAPI data has the "settlement size/security" `+`s on the name:

```
    "name": "Florianz Deposits ++",
```

Whereas, of course, the Journal correctly has only the name:

```
"StationName":"Florianz Deposits",
```
(Docked event)

I'll have a think about how much I want to implement a workaround for this.

In the meantime please [open an issue](https://issues.frontierstore.net/report-issue?step=1) with Frontier about this, including this same Journal and that 21.10 CAPI data file.  Be sure to tell them that it causes an issue because the CAPI `lastStarPort`->`name` doesn't match the StatioName in the Journal `Docked` event.

**Comment by Athanasius** [comment]
This not only affects the `/profile` lastStarport but also the `/market` name field and the same on `/shipyard`.

[Athanasius-HIP 41526-Florianz Deposits-CAPI-rawoutputs.zip](https://github.com/EDCD/EDMarketConnector/files/6816263/Athanasius-HIP.41526-Florianz.Deposits-CAPI-rawoutputs.zip)

**Comment by outsidefactor** [comment]
I will submit an issue on the FDev tracker.

**Comment by outsidefactor** [comment]
Reported as
[Issue 40706](https://issues.frontierstore.net/issue-detail/40706)

**Comment by Athanasius** [comment]
Fix is in 5.1.3
